src/pages/uxp-api/reference-js/Global Members/Data Transfers/fetch.md

```diff
@@ -46,7 +43,7 @@ From UXP v7.4.0 onwards `permissions.network.domains` does not support wildCards
 | [init] | `Object` | `(Optional)` Custom settings for a HTTP request. |
 | [init.method] | `string` | HTTP request method. The default value is "GET". |
 | [init.headers] | `Headers` | HTTP request headers to add. |
-| [init.body] | `string` \| `ArrayBuffer` \| `TypedArray` \| `Blob` \| `FormData` \| `ReadableStream` \| `URLSearchParams` | Body to add to HTTP request. |
-| [init.credentials] | `string` | Indicates whether to send cookies. The default value is "include". Possible values and functions are as follows: <ul style="list-style: none;">  <li>"omit" : cookies are NOT sent.</li><li>"include" : cookies are sent.</li> </ul> |
+| [init.body] | `string` \| `ArrayBuffer` \| `TypedArray` \| `Blob` \| `FormData` \| `URLSearchParams` | Body to add to HTTP request. |
```

Comment:
If we dont support `ReadableStream`, we should call that out as a breaking change.

Reply:
This is just a documentation correction.